HW_3/myproject/homeapp/templates/homeapp/index.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'homeapp/base.html' %}
+
+{% block content %}
+<h1>Заказы за последнюю неделю</h1>
+{{ order_7 }}
```

Comment:
И тут чуть поинтересней
```suggestion
<ul>
    {% for order in order_7 %}
    <li>
        {{ order }}
    </li>
    {% endfor %}
</ul>
```